article.md

```diff
@@ -253,6 +253,94 @@ Depuis le début, il est possible d'utiliser n'importe quels objets comme annota
 
 ## *Unpacking* généralisé -- PEP 448
 
+L'*unpacking* est l'opération qui permet de passer un itérable comme arguments de fonction comme si son contenu était passé élément par élément grace à l'opérateur `*` ou de passer un dictionnaire comme arguments nommés avec l'opérateur `**` :
```

Comment:
Peut-être serait-il judicieux de décrire ce qu'*est* l'*unpacking* : 

L'*unpacking* est TODO. Il permet de passer un itérable comme arguments de fonction ...